rest/src/db/CatapultDb.js

```diff
@@ -304,24 +305,31 @@ class CatapultDb {
 	 * `pageSize` and `pageNumber`. 'sortField' must be within allowed 'sortingOptions'.
 	 * @returns {Promise.<object>} Transactions page.
 	 */
-	transactions(group, filters, options) {
+	async transactions(group, filters, options) {
 		const sortingOptions = { id: '_id' };
 
-		const buildAccountConditions = () => {
-			if (undefined !== filters.address)
-				return { 'meta.addresses': Buffer.from(filters.address) };
+		const buildAccountConditions = async () => {
+			// Check multisig graph if address is used in search criteria for cosigning,
+			// Then, show transactions for other cosigers.
+			if (undefined !== filters.address) {
+				const multisigEntries = await new MultisigDb(this).multisigsByAddresses([filters.address]);
+				const buffers = [];
 
+				if (multisigEntries.length && multisigEntries[0].multisig.multisigAddresses.length)	{
+					buffers.push(...multisigEntries[0].multisig.multisigAddresses).map(address => address.buffer);
```

Comment:
change with:

const buffers = multisigEntries[0].multisig.multisigAddresses.map(address => address.buffer);

Current line has the parentesis in the wrong place